docs/reference/migration/migrate_9_0.asciidoc

```diff
@@ -0,0 +1,314 @@
+[[migrating-9.0]]
+== Migrating to 9.0
```

Comment:
We will end fighting abit with the automation. So even if i clean this up now, it will just come back the next time someone generates the release notes. I think we will need to fix this is in the automation, or just go manual, or something in-between. I would prefer to address in a follow up since the changes here are intended to mechanical. I will add a comment to this file (and others) something to the effect that this is g

Reply:
I updated this PR with a comment for these generated file that links to an internal issue for us to follow up with. I also commented out the content created by the automation. While no one is probably looking at the v9 release notes (it's not released yet) it is better to have no information than wrong information.

Reply:
💯